docs/install/windows.md

```diff
@@ -56,7 +53,7 @@ reg add HKCU\Console /v VirtualTerminalLevel /t REG_DWORD /d 1
 .\prysm.bat beacon-chain
 ```
 
-It is also recommended to include the `--p2p-host-ip` and `--min-sync-peers 7` flags to improve peering. For advanced users that desire standard debugging tools found in the Busybox base image, append a `--debug` flag to enable them.
+> Advanced users: Not getting enough peers?  Refer to the [improve P2P connectivity](/docs/prysm-usage/p2p-host-ip) section of this documentation for tips on network configuration.
```

Comment:
I'm not really sure saying it's for advanced users gives the right impression, as it may very well be beginners as well that need to change firewall settings